packages/openactive-broker-microservice/app.js

```diff
@@ -1025,6 +1027,28 @@ app.post('/assert-unmatched-criteria', function (req, res) {
   }
 });
 
+// put our endpoint here
+app.get('/sample-requests', function (req, res) {
+  // Get random opportunity ID
+  const opportunity = req.body;
+  const opportunityType = detectOpportunityType(opportunity);
+  const sellerId = opportunity.superEvent?.organizer || opportunity.facilityUse?.provider;
```

Comment:
This should make use of: https://github.com/openactive/openactive-test-suite/blob/6f477cfc64f318ba198d93c5f83db31daa38d1cf/packages/openactive-broker-microservice/app.js#L943-L950



```suggestion
  const sellerId = detectSellerId(opportunity);
```